### PR TITLE
CD 3819 : Changed the dropdown to button

### PR DESCRIPTION
**Pull request by jagadeshpaladugula**
Please be aware that we are not actively looking for feature contributions. The truth is that it's extremely difficult for someone outside SonarSource to comply with our roadmap and expectations. Therefore, we typically only accept minor cosmetic changes and typo fixes. If you would like to see a new feature, please create a new thread in the forum ["Suggest new features"](https://community.sonarsource.com/c/suggestions/features).

With that in mind, if you would like to submit a code contribution, make sure that you adhere to the following guidelines and all tests are passing (Travis build is executed for each pull request):

- [ ] Please explain your motives to contribute this change: what problem you are trying to fix, what improvement you are trying to make
- [ ] Use the following formatting style: [SonarSource/sonar-developer-toolset](https://github.com/SonarSource/sonar-developer-toolset#code-style)
- [ ] Provide a unit test for any code you changed
- [ ] If there is a [JIRA](http://jira.sonarsource.com/browse/SONAR) ticket available, please make your commits and pull request start with the ticket ID (SONAR-XXXX)

We will try to give you feedback on your contribution as quickly as possible.

Thank You!
The SonarSource Team

### CODE REVIEW

File: server/sonar-web/src/main/js/apps/projects/components/ProjectCreationMenu.tsx [hunk 1]
```diff
@@ -93,43 +95,30 @@ export class ProjectCreationMenu extends React.PureComponent<Props, State> {
...
-  render() {
-    const { className, currentUser } = this.props;
-    const { boundAlms } = this.state;
+  handleProjectCreate(){
+    let url = window.location.href;
+    let newUrl = "";
+    if(url.endsWith("/projects")){
+      newUrl = url.replace("projects", "extension/developer/projects");
+    }
+    window.location.href = newUrl;
```
Comment (borisbsu) [inline server/sonar-web/src/main/js/apps/projects/components/ProjectCreationMenu.tsx:104]: @jagadeshpaladugula 
Why you don't use the React Router? Please re-do.

Comment (jagadeshpaladugula) [reply server/sonar-web/src/main/js/apps/projects/components/ProjectCreationMenu.tsx:104]: Ok. Let me try in that way

**Review by anurag-ar — COMMENTED** [review]
Hi Jagadesh, If you can provide a screenshot of how the change would look, it will be better

**Comment by jagadeshpaladugula** [comment]
> Hi Jagadesh, If you can provide a screenshot of how the change would look, it will be better

Screenshot 2023-07-11 at 5 47 09 AM

**Review by anurag-ar — APPROVED** [review]

**Pull request merged**